src/backends/kubernetes/kubernetes_backend.py

```diff
@@ -557,6 +597,8 @@ def deploy_pods(self, warnet):
             bitcoind_pod = self.create_pod_object(
                 tank, bitcoind_container, self.get_pod_name(tank.index, ServiceType.BITCOIN)
             )
+            prometheus_container = self.create_prometheus_container(tank)
```

Comment:
in a4d4ee1319cd135f519210a2be75924cd05dc5cd "Add prometheus to logging stack":

We are only scraping selected containers per the filter on `tank.exporter`:

```python
    def apply_prometheus_scrape_configs(self, tanks):
        for tank in tanks:
            if not tank.exporter
```

Would it not make more sense to not add a prometheus container to them under the same condition?

Reply:
Added this